css/style.css

```diff
@@ -0,0 +1,227 @@
+}
+.nav-bar ul .nav-ipod{
+    background: url(../assets/globalnav_text.svg);
+    background-position: 50% -88px;
+}
+.nav-bar ul .nav-ipad{
+    background: url(../assets/globalnav_text.svg);
+    background-position: 50% -148px;
+}
+.nav-bar ul .nav-itunes{
+    background: url(../assets/globalnav_text.svg);
+    background-position: 50% -178px;
+}
+.nav-bar ul .nav-support{
+    background: url(../assets/globalnav_text.svg);
+    background-position: 50% -208px;
+}
+.nav-bar .search-bar{
+    display: inline-block;
+    padding: 0;
+    position: absolute;
+    top: 0;
+    right: 0;
+    box-shadow: none;
+    display: inline;
+    padding-top: 10px;
+    min-width: 94px;
+}
+.nav-bar .search-bar input{
+    transition: 0.3s;
+    position: absolute;
+    right: 15px;
+    top: 65px;
+    background: transparent;
+    padding: 5px 7px;
+    border-radius: 15px;
+    outline: 0;
+    box-shadow: inset 0 2px 2px 0 #333,
+    inset 0 -0.5px 1px 0 #eee;
+    border: none;
+    height: 24px;
+    width: 100px;
+}
+.main-image{
+    background: url(../assets/hero.jpg) no-repeat bottom;
+    height: 647px;
+    text-align: center;
+    color: #fff;
+}
+.body-text {
+    margin-top:0;
+    text-align: center;
+    color: #eee;
+    font-family: Roboto, "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+    margin-bottom: 250px;
+}
+.body-text h1{
+    font-weight: 100;
+    font-size: 400%;
+    margin-bottom: 0;
+    margin-top:0;
+    padding-top: 200px;
+}
+.body-text p{
+    font-size: 200%;
+    margin-top:0;
+}
+.body-text a{
+    font-size: 150%;
+    margin-top:0;
+    text-decoration: none;
+    color: #eee;
+}
+.body-text a:hover{
+ text-decoration: underline;
+}
+.body-ads{
+    
+    height: 200px;
+    width: 1440px;
+    box-sizing: initial;
+    
+
+    
+    
+}
+.body-ads .ad1{
+    background: url(../assets/ad1.jpg) no-repeat;
+    float: left; 
+    height: 200px;
+    width: 360px;
+    display: inline-block;
+}
+.body-ads .ad2{
+    background: url(../assets/ad2.jpg) no-repeat;
+    float: left; 
+    height: 200px;
+    width: 360px;
+    display: inline-block;
+}
+.body-ads .ad3{
+    background: url(../assets/ad3.jpg) no-repeat;
+    float: left; 
+    height: 200px;
+    width: 360px;
+    display: inline-block;
+}
+.body-ads .ad4{
+    background: url(../assets/ad4.jpg) no-repeat;
+    float: left; 
+    height: 200px;
+    width: 360px;
+    display: inline-block;
+    padding-bottom: 30px;
+}
+.footer-links{
+    display:inline;
+}
+.footer{
+    margin-top: 30px;
+    margin-left: 50px;
+    color: gray;
+    display:inline;
+    font-size: 76%;
+    text-decoration: none;
+    box-sizing: initial;
+    letter-spacing: 1px;
+    
+}
+.footer-links li{
+    display:inherit;
+    
+}
+.footer span{
+    margin-left: 50px;
+    
+}
+.footer a:hover{
+ text-decoration: underline;
+}
+.footer-bottom{
+    display:block;
+    margin-left:50px;
+}
+.footer a{
+    text-decoration: none;
+    color:rgb(106, 106, 218);
```

Comment:
I think the color doesn't match the original

Reply:
Can you make the font [sans-serif ](https://www.webopedia.com/TERM/S/sans_serif.html) so it matches the original?